frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/EditFieldDrawer/edit-fieldtype/EditDate/EditDate.tsx

```diff
@@ -216,6 +276,50 @@ export const EditDate = ({ field }: EditDateProps): JSX.Element => {
           {get(errors, 'dateValidation.customMinDate.message')}
         </FormErrorMessage>
       </FormControl>
+      <Stack>
+        <FormControl isReadOnly={isLoading}>
+          <Toggle
+            {...register('addParticularDayRestriction')}
+            label="Customize days of the week"
+            description="Checking a day will disable all the same days in the calendar"
+          />
+        </FormControl>
+        {getValues('addParticularDayRestriction') ? (
```

Comment:
should we use `watch` over `getValues`? [Docs](https://react-hook-form.com/api/useform/getvalues#:~:text=An%20optimized%20helper%20for%20reading%20form%20values.%20The%20difference%20between%20watch%20and%20getValues%20is%20that%20getValues%20will%20not%20trigger%20re%2Drenders%20or%20subscribe%20to%20input%20changes.) seem to imply the value returned by `watch` should be used in the render conditional over `getValues`